aceitar_termo_handler.go

```diff
@@ -46,7 +48,30 @@ func newAceitarTermoFormHandler(dbClient *db.Client) echo.HandlerFunc {
 		}
 		email := claims["email"]
 		fmt.Println(email)
-		// TODO: save the acceptance of the terms in the DB
+		foundCandidate, err := dbClient.GetCandidateByEmail(email, year)
+		if err != nil {
+			log.Printf("failed find candidate on DB, error %v\n", err)
+			switch {
+			case err != nil && err.(*exception.Exception).Code == exception.NotFound:
+				return c.Render(http.StatusOK, "atualizar-candidato-success.html", map[string]interface{}{
+					"ErrorMsg": fmt.Sprintf("Não encontramos um cadastro de candidatura através do email %s. Por favor verifique se o email está correto.", email),
+					"Success":  false,
+				})
+			case err != nil:
+				return c.Render(http.StatusOK, "atualizar-candidato-success.html", map[string]interface{}{
+					"ErrorMsg": "Erro inesperado. Por favor, tente novamente mais tarde.",
+					"Success":  false,
+				})
+			}
+		}
+		foundCandidate.AcceptedTerms = time.Now()
```

Comment:
Forçar um timezone para garantir que temos consistência de tempo. Por exemplo, sempre garantir que estamos pegando o tempo em UTC